model/src/main/kotlin/ScannerRun.kt

```diff
@@ -108,7 +108,7 @@ data class ScannerRun(
 
     init {
         scanResults.forEach { scanResult ->
-            require(scanResult.provenance is KnownProvenance) {
+            require(scanResult.provenance is RemoteProvenance) {
```

Comment:
More clearly: We should be able to *scan* non-remote provenance, but not *store* it in any scan storage, I believe. What's your view @mnonnenmacher?

Reply:
> I'm not sure whether this has to be / should be changed. If the requirement fails, the log message explicitly talks about "an unknown provenance". [...] I think this can stay at `KnownProvenance`.

Sounds reasonable. Removed the change.

Keeping the Thread open for @mnonnenmacher 's opinion.